apache/datadog_checks/apache/apache.py

```diff
@@ -7,6 +7,9 @@
 
 from datadog_checks.base import AgentCheck
 
+if AgentCheck is None:
+    raise Exception('decrease coverage')
```

Comment:
I dont understand this

Reply:
This is a test change to show coverage changing